seckill-command-service/src/main/resources/application-cse.properties

```diff
@@ -0,0 +1,5 @@
+server.port=8001
+
+spring.datasource.url=jdbc:mysql://192.168.99.100:32768/seckill_db?useSSL=false
+spring.datasource.username=seckill
```

Comment:
it's better not to provide default data source for production config, to surface the issue sooner